src/relations/postgresql_provider.py

```diff
@@ -190,6 +199,13 @@ def update_endpoints(self, event: DatabaseRequestedEvent = None) -> None:
                 read_only_endpoints,
             )
 
+    def _check_multiple_endpoints(self) -> bool:
+        """Checks if there are relations with other endpoints."""
+        relation_names = {relation.name for relation in self.charm.client_relations}
```

Comment:
@marceloneppel I fixed it, like `set`
or use the implementation from `src/relations/db.py's` `_check_multiple_endpoints` ?